supplier_app/migrations/0001_initial.py

```diff
@@ -66,11 +68,12 @@ class Migration(migrations.Migration):
             name='TaxPayerArgentina',
             fields=[
                 ('taxpayer_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='supplier_app.TaxPayer')),
-                ('cuit', models.CharField(max_length=200)),
-                ('comments', models.CharField(max_length=200)),
-                ('payment_type', models.CharField(max_length=200)),
+                ('cuit', models.CharField(max_length=20)),
+                ('payment_type', models.CharField(choices=[('BANK', 'Bank')], max_length=20)),
+                ('payment_term', models.IntegerField(choices=[(15, '15 days'), (30, '30 days')], default=None)),
```

Comment:
The default value should be `None` or should we ask our PM?

Reply:
For now the default value is 15 days, so I change that